.github/workflows/create-pre-release.yml

```diff
@@ -0,0 +1,86 @@
+---
+name: "Create a pre-release"
+
+# This action will run when it is triggered manually
+on:
+  workflow_dispatch:
+    inputs:
+      tagName:
```

Comment:
As mentioned above, do we follow a strict naming convention for tag names?

Reply:
Yes, see [my reply above](https://github.com/Automattic/woocommerce-payments/pull/4925#discussion_r998013653)